states/states.c

```diff
@@ -8,7 +8,7 @@
 extern char **environ;
```

Comment:
What is this variable ? It's not the first time I see it.
Is it the environment ?

Reply:
Indeed it is: `man 7 environ` (http://man7.org/linux/man-pages/man7/environ.7.html)